hiv-benchmarking/7_hiv05_comparison_ratios.ipynb

```diff
@@ -0,0 +1,1376 @@
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "5ae2b5da-45ed-4f15-850f-c363e4aab26a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.2465642683912692"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "publication_nodes_in_ensemble_ratio = len(both_pub_ensemble)/len(all_ensemble_proteins)\n",
+    "publication_nodes_in_ensemble_ratio"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "363b387b-1f5a-4eeb-9808-67653083cd6e",
+   "metadata": {},
+   "source": [
+    "##### TOTAL ratio of nodes in ensemble pathway that are also in publication pathway"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "5fc29a55-3d47-4a2e-8d8e-66fcb00c5b4d",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.8764367816091954"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ensemble_nodes_in_publication_ratio = len(both_pub_ensemble)/len(all_pub_proteins)\n",
+    "ensemble_nodes_in_publication_ratio"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8eb4e036-e935-42e1-afb0-607ca37612d5",
+   "metadata": {},
+   "source": [
+    "##### MEDIUM CONFIDENCE -> ratio of nodes in publication pathway that are also in ensemble pathway:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "209ff75b-477b-4bc9-8463-bbac9c51ab8d",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.2631578947368421"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "medconf_publication_nodes_in_ensemble_ratio = len(medconf_both_pub_ensemble)/len(medconf_all_ensemble_proteins)\n",
+    "medconf_publication_nodes_in_ensemble_ratio"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9bef467d-0d4d-47d0-8540-33ea37848284",
+   "metadata": {},
+   "source": [
+    "##### MEDIUM CONFIDENCE -> ratio of nodes in ensemble pathway that are also in publication pathway"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "id": "927aee7e-ba27-4cff-8ba5-a9a005fceb17",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.3879310344827586"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "medconf_ensemble_nodes_in_publication_ratio = len(medconf_both_pub_ensemble)/len(all_pub_proteins)\n",
+    "medconf_ensemble_nodes_in_publication_ratio"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8bb9115a-9d90-4ea0-a9a1-d642f8968399",
+   "metadata": {},
+   "source": [
+    "##### LOW CONFIDENCE -> ratio of nodes in publication pathway that are also in ensemble pathway:"
```

Comment:
All the code after this point seems to be empty. Could you remove up the code blocks and associated markdowns if they are supposed to be empty.